sdcm/sct_events/__init__.py

```diff
@@ -18,6 +18,7 @@
 
 
 class Severity(enum.Enum):
+    STARRED = -1  # this should only be through test case configuration!!!
```

Comment:
what `STARRED` means ?

Reply:
that's exactly what I meant to ask, it is not related to the term severity

Reply:
starred, pinned, marked...

Reply:
like in your Gmail, where you are able to start messages

Reply:
changed to `debug` as it was agreed on the meeting couple of days ago